lib/network/modules/EdgesHandler.js

```diff
@@ -6,14 +6,13 @@ var Edge = require("./components/Edge").default;
 var Label = require("./components/shared/Label").default;
 
 /**
- * @class EdgesHandler
+ * @param {Object} body
+ * @param {Array.<Image>} images
+ * @param {Array.<Group>} groups
  */
 class EdgesHandler {
   /**
-   * @param {Object} body
-   * @param {Array<Image>} images
-   * @param {Array<Group>} groups
-   * @constructor EdgesHandler
+   * @inheritDoc
```

Comment:
No parent class, I don't see the point of this tag here.
This also for further occurrences.